### DISCUSSION TITLE
Questions about any updates of methods to embed LinearGenomeView

### DISCUSSION THREAD

**Discussion by ghost** [GMOD/jbrowse-components#1436] [Q&A] [answered]
In the tutorial of [BCC2020](https://jbrowse.org/jb2/docs/bcc2020_embedding_jbrowse_01_getting_started/), a method by including a single js file was adopted.
Is there any NPM package available?

In #1313, I found the way to embed JBrowse LinearGenomeView by React components.
That is a pretty nice work, how soon could we see the package to be published on NPM?

**Comment by ghost** [comment]
Have seen updates  #1469  #1502 , thanks for working on the new release #1508 .

**Comment by garrettjstevens** [comment, marked as answer]
Thanks for your patience as we work on getting this published. There is a version published to NPM now. There is a small bug that requires adding one more package to your dependencies as a workaround, which is described here: https://github.com/GMOD/jbrowse-components/issues/1502#issuecomment-737424279

Also, you can see a simple example here: https://github.com/GMOD/jbrowse-components/blob/edc8e0adb3caca12af328394c72c11a81fde4557/products/jbrowse-react-linear-genome-view/docs/example.md
